kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/AdminManager.java

```diff
@@ -256,7 +256,7 @@ public void deleteTopic(String topicToDelete,
                             Consumer<String> successConsumer,
                             Consumer<String> errorConsumer) {
         admin.topics()
-                .deletePartitionedTopicAsync(topicToDelete)
+                .deletePartitionedTopicAsync(topicToDelete, true, true)
```

Comment:
```suggestion
                .deletePartitionedTopicAsync(topicToDelete, true)
```
Use {@link Topics#deletePartitionedTopicAsync(String, boolean)} instead because parameter `deleteSchema` is always true

Reply:
Ok, let's keep use `deletePartitionedTopicAsync(topicToDelete, true, true)`.